overlord/servicestate/quota_handlers_test.go

```diff
@@ -58,6 +58,37 @@ func (s *quotaHandlersSuite) SetUpTest(c *C) {
 	s.AddCleanup(r)
 }
 
+// mockMixedQuotaGroup creates a new quota group mixed with the provided snaps and
+// a single sub-group with the same name appended with 'sub'. The group is created with
+// the memory limit of 1GB, and the subgroup has a limit of 512MB.
```

Comment:
thanks for this comment, something slightly more might be nice like "This sort of mixed quota group is no longer supported but was briefly available while the feature was experimental"

Reply:
It's a good detail to have, I'll add it in!